CorePayments/src/main/java/com/paypal/android/corepayments/SecureTokenServiceAPI.kt

```diff
@@ -0,0 +1,49 @@
+package com.paypal.android.corepayments
+
+import android.util.LruCache
+
+class SecureTokenServiceAPI internal constructor(
```

Comment:
Can we document that this can be found at `go/ppass` under "Secure Token Service (v1)"? just so future devs know too that these API classes are 1:1 with ppass services